packages/app-utils/src/components/ConnectionBootstrap.tsx

```diff
@@ -104,21 +186,43 @@ export function ConnectionBootstrap({
     [objectFetcher]
   );
 
-  if (connection == null || error != null) {
+  function handleRefresh(): void {
+    log.info('Refreshing application');
+    window.location.reload();
+  }
+
+  if (isShutdown) {
     return (
       <LoadingOverlay
-        data-testid="connection-bootstrap-loading"
-        isLoading={connection == null}
```

Comment:
This `testid` should not be removed. It makes the couple assertions for tests unable to find this.